docs/Using-Fleet/REST-API.md

```diff
@@ -918,6 +918,7 @@ None.
     "jira": null
   },
   "mdm": {
+    "enabled_and_configured": false,
```

Comment:
I think it should also be documented in the `PATCH /appconfig` response example below (and I just noticed `"apple_bm_terms_expired": false,` is missing from the example here if you could add it too?) :pray:

Reply:
great catch, thank you! Fixed in [b955a68](https://github.com/fleetdm/fleet/pull/9575/commits/b955a689cb70b71d74774016bd90ab5534e82020)